seaice/pipeline/config.py

```diff
@@ -0,0 +1,8 @@
+"""This config file is used to ensure the correct GDAL and PROJ libraries are used. We set the GDAL_DATA and PROJ_LIB environment variables. The behavior of anaconda-project with regards to these variables is a known issue: https://github.com/Anaconda-Platform/anaconda-project/issues/349. Import this config file before other geospatial imports (rasterio, pyproj, etc.)."""
+
+import os
+
+if os.getenv("GDAL_DATA") != "":
```

Comment:
Is this intending to compare to empty string, or unset?

Reply:
This is comparing to see if there is no library available to conda. [Ticketed](https://github.com/ua-snap/pipeline-template/issues/3) over in the pipeline template repo.